### PR TITLE
Adding IndiGo icon

### LINKED ISSUE #9141: IndiGO
### Brand Name

IndiGO

### Website

https://www.goindigo.in/

### Popularity Metric

Global Rank: 3,771   https://www.similarweb.com/website/goindigo.in/#overview

### Official Resources for Icon and Color

Unknown

### Additional Comments

Unknown

### Contributing

No

### PR DESCRIPTION
**Pull request by CodeWithShipraJain**
**Issue:** closes #9141

**Similarweb rank:** [3,771](https://www.similarweb.com/website/goindigo.in/#overview)
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5870,6 +5870,11 @@
             "hex": "003A9B",
             "source": "https://indeed.design/resources"
         },
+        {
+            "title": "IndiGo",
+            "hex": "FFFFFF",
```
Comment (adamrusted) [inline _data/simple-icons.json:5875]:
```suggestion
            "hex": "09009B",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/indigo.svg
[the remark this answers is not in the document]
Comment (CodeWithShipraJain) [reply icons/indigo.svg:1]: @adamrusted yes it is caused due to SVGO precision, I have updated the image with precision value 4.

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [reply _data/simple-icons.json:5875]: Happy with the SVG now, but I'd still go with the hex change suggested here. Once that's done - we can get this merged!

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @CodeWithShipraJain 🎉

**Pull request merged**